lib/fog/vmfusion.rb

```diff
@@ -9,6 +9,6 @@ module Vmfusion
   end
 
   module Compute
-    autoload :Vmfusion, 'fog/compute/vmfusion'
+    autoload :Vmfusion, File.expand_path('../compute/vmfusion', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.